src/05-derivatives/03-imaging.md

```diff
@@ -0,0 +1,368 @@
+| Type         | RECOMMENDED. Short identifier of the mask. Reserved values: `Brain` - brain mask, `Lesion` - lesion mask, `Face` - face mask, `ROI` - ROI mask |
+| Resolution   | REQUIRED if `res` is present. String, or [object][] mapping labels to strings. Specifies the interpretation of the resolution keyword.         |
+| Density      | REQUIRED if `den` is present. String, or [object][] mapping labels to strings. Specifies the interpretation of the density keyword.            |
+
+Examples:
+
+```Text
+func_loc/
+    sub-001/
+        func/
+           sub-001_task-rest_run-1_space-MNI305_desc-PFC_mask.nii.gz
+           sub-001_task-rest_run-1_space-MNI305_desc-PFC_mask.json
+```
+
+```Text
+manual_masks/
+    sub-001/
+        anat/
+            sub-001_desc-tumor_mask.nii.gz
+            sub-001_desc-tumor_mask.json
+```
+
+## Segmentations and parcellations
+
+Common JSON metadata fields:
+
+| **Key name** | **Description**                                                                                                                        |
+| ------------ | -------------------------------------------------------------------------------------------------------------------------------------- |
+| Manual       | OPTIONAL. Boolean. Indicates if the segmenation was performed manually or via an automated process                                     |
+| Atlas        | OPTIONAL. Which atlas (if any) was used to derive the segmentation.                                                                    |
+| Resolution   | REQUIRED if `res` is present. String, or [object][] mapping labels to strings. Specifies the interpretation of the resolution keyword. |
+| Density      | REQUIRED if `den` is present. String, or [object][] mapping labels to strings. Specifies the interpretation of the density keyword.    |
+
+### Discrete Segmentations
+
+Discrete segmentations of brain tissue represent each tissue class with a unique
+integer label in a 3D volume. See [Anatomical Labels](#anatomical-labels) for interpretation how
+integer values map to tissue classes.
+
+Template:
+
+```Text
+<pipeline_name>/
+    sub-<participant_label>/
+        anat|func|dwi/
+            <source_entities>[_space-<space>][_res-<label>][_den-<label>]_dseg.nii.gz
+```
+
+Example:
+
+```Text
+pipeline/
+    sub-001/
+        anat/
+            sub-001_space-orig_dseg.nii.gz
+            sub-001_space-orig_dseg.json
+```
+
+A segmentation could be a binary mask that functions as a discrete `label` for a
+single structure. In this case, the label key must be used to specify the
+corresponding structure. For example:
+
+```Text
+pipeline/
+    sub-001/
+        anat/
+            sub-001_space-orig_label-GM_dseg.nii.gz
+```
+
+See [Anatomical Labels](#anatomical-labels) for reserved key values for `label`.
+
+### Probabilistic Segmentations
```

Comment:
Might be late to the party, but I'm nevertheless going to push here and see if something breaks.
(Please link if there are previous conversations I've failed to find)

I don't like "Probabilistic segmentations".

Take for example FSL's `fast`. The images produced by that command do not (ideally) represent *probabilities* of an individual voxel containing one tissue or the other, but *volume fractions* where voxels can contain multiple tissues. Another example is if I take a surface-based representation of some structure, and directly map that information to a voxel image by calculating the fraction of the volume of each voxel that resides within that closed surface (one could argue that such is equivalent to the probability of a randomly-selected point within that voxel residing within the structure, but it's not a direct & intuitive interpretation; describing such as volume fractions makes a lot more sense).

The antonym of "discrete" is "continuous"; some, *but not all*, continuous segmentation data represent probabilities. If the suffix is to encode the *nature* of the data, rather than the *interpretation* of such, "continuous" would be more general.

Reply:
On top of my head, I believe FSL fast produces probability maps either way. The tool gives you the posterior (probability) map of each tissue from the mixture model. The PVEs output (volume fraction) are just the output of the full model (including the regularization of the MRF). I agree there's some implicit spatial meaning to "volume fraction" (in that it suggests the concentration across the voxel may not be uniform) that seems less obvious for a probability. In addition, volume fraction is not applicable to 1D and implies some gymnastics in 2D. Probability seems more general.

That said, I don't particularly love `_probseg`, but I would probably love `_cseg` or `_contseg` less.

I worry more about the `seg` component here because it implies an assignment of the samples to a class with some implicit MAP criterion for `_probseg` and possibly normalization of the probabilities. In the discrete case, "seg" seems to limit the of classes a sample can be assigned to just one. Maybe, `_labels` instead of `_dseg` and `_prob` instead of `_probseg` would be clearer?

Either way, I think this conversation surfaces the need for clarity in describing these types of derivatives (i.e., can I have several label assignments for one sample in a `_dseg`?, should `_probseg` be always in the 0 .. 1 range? Should it be normalized to sum 1 along the last dimension?) rather than the appropriateness of the suffixes.

Reply:
If we want to reopen this discussion, I think we should move segmentations out of common derivatives and move it into the structural derivatives BEP.

Reply:
These names have been around for a good amount of time. I don't see an advantage to bikeshedding names at this late date, and there haven't been fleshed-out proposals for alternative data structures.